pkgs/top-level/lua-packages.nix

```diff
@@ -215,6 +215,30 @@ let
     '';
   };
 
+  luaiconv = buildLuaPackage rec {
+    name = "luaiconv-${version}";
+    version = "7";
+    src = fetchFromGitHub {
+      owner = "ittner";
+      repo = "lua-iconv";
+      rev = "e8d34024a6b185a759733915f116cc5588550261";
+      sha256 = "0rd76966qlxfp8ypkyrbif76nxnm1acclqwfs45wz3972jsk654i";
+    };
+
+    buildInputs = [];
+    meta = {
+      platforms = stdenv.lib.platforms.unix;
+      license = stdenv.lib.licenses.mit;
+      description = "Lua bindings for POSIX iconv";
+      maintainers = [ maintainers.richardipsum ];
+    };
```

Comment:
Move meta to the end, use `with stdenv.lib` and `with maintainers`, add homepage.

Reply:
Will do

Reply:
done